kafka/manager.go

```diff
@@ -70,13 +72,23 @@ func NewManager(cfg ManagerConfig) (*Manager, error) {
 	if err != nil {
 		return nil, fmt.Errorf("kafka: failed creating kafka client: %w", err)
 	}
-	m := &Manager{
+	if cfg.MeterProvider == nil {
+		cfg.MeterProvider = otel.GetMeterProvider()
+	}
+	meter := cfg.MeterProvider.Meter("github.com/elastic/apm-queue/kafka")
+	deleted, err := meter.Int64Counter("topics.deleted.count",
+		metric.WithDescription("The number deleted topics"),
+	)
```

Comment:
WDYT about having the `outcome` dimension for both `topics.{created, deleted}.count` and recording failures (excluding the topic already exists case) to create/delete topics?